tech/languages/python/multiple-pythons.md

```diff
@@ -187,6 +188,58 @@ Type "help", "copyright", "credits" or "license" for more information.
 To learn more about virtualenvs, visit
 [The Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
 
+### Python 2.6, Jython
+
+The versions of virtualenv and tox packaged in Fedora do not support Python 2.6
+and Jython, because they are extremely old. However, there is a trick on how to
```

Comment:
Same problem here.

Reply:
`virtualenv and tox packages in Fedora are incompatible with the following interpreters:`
 - Python 2.6 and Jython